src/index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="day day--day-1 day--start-day_mon"></div>
+      <div class="day day--day-3" ></div>
+      <div class="day day--day-4" ></div>
+      <div class="day day--day-5" ></div>
```

Comment:
Перечитай уважно завдання
класи з модифікаторами мають стояти тільки в блоці calendar, кожен день нумерувати не треба
має вийти щось таке
```suggestion
    <main class="main">
      <div class="calendar calendar--start-day--tue calendar--month-length--29">
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
        ...
      </div>
      
```